src/transport/net_ib.cc

```diff
@@ -1329,4 +1362,3 @@ ncclNet_t ncclNetIb = {
   ncclIbCloseRecv,
   ncclIbCloseListen
 };
-
```

Comment:
remove this deletion (irrelevant and unnecessary)

Reply:
This seems to be a styling preference for our linter... Thanks for pointing this out! Reverted the change.